src/zjs_aio.json

```diff
@@ -0,0 +1,10 @@
+{
+    "module":"aio",
+    "require":"aio",
+    "targets":["arduino_101"],
+    "zephyr_conf": {
+        "all":["CONFIG_ADC=y"]
+    },
+    "zjs_config": ["-DBUILD_MODULE_AIO"],
```

Comment:
... and here you have space after `:`...

Would be nice to have a small consistency with this stuff (apparently, with space after `:`).